### DISCUSSION TITLE
Order entities' integration by device.

### DISCUSSION THREAD

**Discussion by Julien-Decoen** [jseidl/magic-areas#263] [Ideas / Feature Requests]
It could be cool to have en device page for each area to see all entities related to the area from Magic Areas :) 

Exemple in the Plant Monitor integration :

<img width="259" alt="Capture d’écran 2022-10-23 à 16 19 13" src="https://user-images.githubusercontent.com/22341879/197397390-ddc50e1a-4bc2-4bf4-9098-20611fc2360a.png">

<img width="1383" alt="Capture d’écran 2022-10-23 à 16 22 40" src="https://user-images.githubusercontent.com/22341879/197397423-5bb23e54-b3ec-4602-94bb-72b89f090a61.png">

**Comment by jseidl** [comment]
You can do that by clicking on the area instance on Magic areas integration page and click "entities".

Device might be cool, although I'm conflicted because MA isn't really a device.

**Comment by jseidl** [comment]
This is actually done in the latest version :D

**Reply by Julien-Decoen** [reply]
Good job ! :)
